src/ray/common/network_util.h

```diff
@@ -14,4 +14,6 @@
 
 #pragma once
 
-bool CheckFree(int port);
+// Check whether the given [port] is available, via attempt to bind a socket to the port.
+// Notice, the check could be non-authentic if there're concurrent port assignments.
```

Comment:
```suggestion
// Notice, the check could be non-authentic if there are concurrent port assignments.
```

Reply:
Hmm what's the difference? chatgpt told me they're the same.